releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,30 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 2.6.0-preview2 - June 7, 2023
+
+### Enhancement - Parameter Caching
+
+Developers using our `ConfigurationFeatureDefinitionProvider` will now benefit from caching of filter parameters. This will improve performance of the library by reducing the number of times a filter's parameters are cast in short time frames, allowing loops to execute up to 100x faster. This change will not affect custom filters by default. For custom filters, the class must implement the `IFilterParametersBinder` interface. Below is an example.
```

Comment:
> allowing loops to execute up to 100x faster

It's somewhat odd to see loops being called out explicitly, given they are a bit of a contrived performance scenario. I'd suggest to have something like "yielding observed performance increase of up to 100x."

Reply:
I wanted to call out that the 100x performance increase is only true if calling IsEnabledAsync repeatedly. As calling it once every 1 second has a negligible performance increase. I suppose "short time frame" indicates that. I'll go with your suggestion.